website/blog/2019-04-12-new-integration-structure.md

```diff
@@ -0,0 +1,23 @@
+---
+author: Paulus Schoutsen
+authorURL: https://twitter.com/balloob
+authorFBID: 297400035
+title: Introducing Integrations
+---
+
+We have finished [the great migration](/blog/2019/02/19/the-great-migration.html). The result will be released in Home Assistant 0.92. This release has been a bit delayed because we had a lot of things to fix! With the migration done, we now consider components and platforms that share the same name to be part of the same integration. Each integration is either a single Python file, or a folder with an `__init__.py`. file.
+
+Home Assistant 0.92 introduces a new [`manifest.json`](/docs/en/creating_integration_file_structure.html) for integrations. This file, which is optional, is used by integrations to specify metadata: name, link to the documentation, dependencies, requirements and code owners. We are exploring leveraging `manifest.json` for additional future features, like tracking breaking changes or allowing custom components to provide config flows and being discovered.
```

Comment:
optional for custom component, but required for "official" component

Reply:
fixed